### PR TITLE
ID localization for administrator cluster - cluster management

### PR DESCRIPTION
**Pull request by ariscahyadi**
This PR is addressing `/docs/tasks/administer-cluster/cluster-management` as mentioned in #22296.

Previously, it was claimed by @phanama but it seems not completed yet.

### CODE REVIEW

File: content/id/docs/tasks/administer-cluster/cluster-management.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+
+Dokumen ini menjelaskan beberapa topik yang terkait dengan siklus hidup sebuah klaster: membuat klaster baru,
+memperbarui Node _control plane_ dan Node pekerja dari klaster kamu,
+melakukan pemeliharaan Node (misalnya pembaruan kernel), dan meningkatkan versi API Kubernetes dari
+klaster yang berjalan.
+
+<!-- body -->
+
+
+## Membuat dan mengonfigurasi klasterter
```
Comment (irvifa) [inline content/id/docs/tasks/administer-cluster/cluster-management.md:16]:
```suggestion
## Membuat dan mengonfigurasi klaster
```

**Review by irvifa — CHANGES_REQUESTED** [review]
There’s typo that I see and it’s minor. Thanks for the contribution 😀

**Comment by ariscahyadi** [comment]
@irvifa and @girikuncoro the issue has been addressed. Please approve!

**Comment by irvifa** [comment]
Thank you!
/lgtm
/approve

**Pull request merged**